code/baseline_utils.py

```diff
@@ -0,0 +1,58 @@
+import colorsys
+import cmath as cm
+from itertools import product
+
+__authors__ = "Anton Gochev, Jaro Habr, Yan Jiang, Samuel Kahn"
+__version__ = "XCS224u, Stanford, Spring 2021"
+
+
+def fourier_transform(hsv_color):
```

Comment:
I suppose we are going to use the fourier transform with the transformes as well. Extracted for reusability.